CoreUtilities/src/au/gov/asd/tac/constellation/utilities/gui/NotifyDisplayer.java

```diff
@@ -219,4 +219,32 @@ public static Optional<ButtonType> displayConfirmationAlert(final String title,
 
         return dialog.showAndWait();
     }
+    
+    
+    /**
+     *
+     * @param title
+     * @param header
+     * @param message
+     * @param okButton
+     * @param cancelButton
+     * @return
+     */
+    public static Optional<ButtonType> displayConfirmationAlert(final String title,
```

Comment:
One more thing, are you possibly able to write a unit test for this function you introduced?